front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/v2/EntityTagsController.groovy

```diff
@@ -48,13 +47,15 @@ class EntityTagsController {
   @RequestMapping(method = RequestMethod.GET)
   Set<EntityTags> tags(@RequestParam(value = "prefix", required = false) String prefix,
                        @RequestParam(value = "ids", required = false) Collection<String> ids) {
-    if (!prefix && !ids) {
+    if (prefix == null && !ids) {
```

Comment:
`clouddriver` is supplying `?prefix=` which is legitimate (_this used to work and was recently broken_).